### PR TITLE
project for review

### PR DESCRIPTION
**Pull request by BregornOriginal**
### In this project, you can add/delete/edit tasks in a taskList.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,76 @@
...
+
+## Authors
+
+👤 **@BregornOriginal**
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (Addisu87) [inline README.md]: Please add the link for issues.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		<section id="list-container">
+			<h1>Today's To Do</h1>
+			<div class="input-group mb-3">
+				<input id="textNewTask" type="text" class="form-control" placeholder="Enter new task" aria-label="Recipient's username" aria-describedby="button-addon2">
+				<button class="btn btn-outline-secondary add-button" type="button" id="button-addon2">Add</button>
+			</div>
+		</section>
+		<div class="clear-button-container">
+			<button class="btn btn-secondary btn-lg bg-success text-white clear-button" type="button">Clear all Completed</button>
+		</div>
```
Comment (Addisu87) [inline src/index.html]: Your code will look better if you use semantic tags.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+const $clearButton = document.querySelector('.clear-button');
+
+export const editText = ($checkBoxContainer, toDo) => {
+  const $newPlaceHolderTask = document.createElement('input');
+  $newPlaceHolderTask.classList.add('checked', 'form-control');
+  $newPlaceHolderTask.setAttribute('type', 'text');
+  $newPlaceHolderTask.value = toDo.textContent;
+  $newPlaceHolderTask.setAttribute('aria-label', 'Text input with checkbox');
+  $checkBoxContainer.replaceChild($newPlaceHolderTask, toDo);
+  $newPlaceHolderTask.addEventListener('keypress', (e) => {
```
Comment (Addisu87) [inline src/index.js:30]: I suggest that you need to remove the dollar sign when you declare a variable. It seems your code looks like a jquery file.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+$addButton.addEventListener('click', (e) => {
+  if ($textNewTask.value) {
+    e.preventDefault();
+    addTask($textNewTask.value);
+    $textNewTask.value = '';
+  }
+});
+
+export const saveStorage = () => {
```
Comment (Addisu87) [inline src/index.js]: Please change your variable names to another one that does not confuse users.

**Review by Addisu87 — APPROVED** [review]
@BregornOriginal 
Your code is so nice and easily understandable, but you need to make some changes.

File: src/index.js (hunk 4 above)
Comment (BregornOriginal) [reply src/index.js]: Thank you, I will change that one for the name store()

File: src/index.html (hunk 2 above)
Comment (BregornOriginal) [reply src/index.html]: Yes! I forgot about putting the main tag in the beginning.

File: README.md (hunk 1 above)
Comment (BregornOriginal) [reply README.md]: Changed!

File: src/index.js (hunk 3 above)
Comment (BregornOriginal) [reply src/index.js:30]: Ok, I will do that. Thank you @Addisu87

**Review by Addisu87 — APPROVED** [review]
You can merge your feature branch. Good job.

**Pull request merged**